src/core/gui/dialog/SettingsDialog.cpp

```diff
@@ -523,15 +523,15 @@ void SettingsDialog::load() {
             break;
     }
 
-    auto activeViewMode = settings->getViewModes().at(VIEW_MODE_FULLSCREEN);
-    bool showFullscreenMenubar = activeViewMode.goFullscreen;
-    bool showFullscreenToolbar = activeViewMode.showToolbar;
-    bool showFullscreenSidebar = activeViewMode.showSidebar;
-    activeViewMode = settings->getViewModes().at(VIEW_MODE_PRESENTATION);
-    bool showPresentationMenubar = activeViewMode.showMenubar;
-    bool showPresentationToolbar = activeViewMode.showToolbar;
-    bool showPresentationSidebar = activeViewMode.showSidebar;
-    bool goPresentationFullscreen = activeViewMode.goFullscreen;
+    auto viewMode = settings->getViewModes().at(ViewModeId::VIEW_MODE_FULLSCREEN);
+    bool showFullscreenMenubar = viewMode.goFullscreen;
```

Comment:
I'm still confused by this line. Shouldn't it be `viewMode.showMenubar`?

Reply:
Oh my, I only now saw what you meant xD Sorry about that. I'll fix this right away...